on-boarding-modules/pytest-examples/test_module1_pytest.py

```diff
@@ -0,0 +1,44 @@
+# Selenium 3.14+ doesn't enable certificate checking
+import pytest
+from selenium import webdriver
+from _pytest.runner import runtestprotocol
+import urllib3
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+
+# @pytest-examples.fixture
+@pytest.mark.skip(reason="Update Sauce Creds")
```

Comment:
Is there some coverage of demonstrating pytest that this is here? I am not sure about conflating pytest demonstration within module1.

Reply:
That flag was for skipping the test for CI, I could just update the `config.yml` to explicitly leave out the test during the build step and then remove the pytest flag here